src/components/Contracts/ContractItem/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import Card from 'antd/es/card';
+import List from 'antd/es/list';
+import Title from 'antd/es/typography/Title';
+import Paragraph from 'antd/es/typography/Paragraph';
+import Markdown from 'markdown-to-jsx';
+import router from '../../../router';
+import { blockchainSelectors, contractsSelectors } from '../../../redux/selectors';
+import Button from '../../Button/Button';
+import ButtonsWrapper from '../ButtonsWrapper';
+import CopyIconWithAddress from '../../CopyIconWithAddress';
+import { deriveAndHideContractTitle } from './utils';
+
+function ContractItem({
+  contractId,
+  creator,
+  data,
+  parties,
+  isOneItem,
+  judgesSignaturesList,
+  partiesSignaturesList,
+  isMyContracts,
+}) {
+  const walletAddress = useSelector(
+    blockchainSelectors.userWalletAddressSelector,
+  );
+  const identitiesContracts = useSelector(
+    contractsSelectors.selectorIdentityContracts,
+  );
+  const isContractSign = judgesSignaturesList?.length > 0 || partiesSignaturesList?.length > 0;
+  const isMeSigned = partiesSignaturesList ? partiesSignaturesList.includes(walletAddress) : false;
+  const isMeSignedAsJudge = judgesSignaturesList ? judgesSignaturesList.includes(walletAddress) : false;
+  const infoContract = [
+    { itemsOrItem: creator, name: 'Creator' },
+    { itemsOrItem: parties, name: 'Parties' },
+    { itemsOrItem: partiesSignaturesList || [], name: 'Parties Signatures' },
+    { itemsOrItem: judgesSignaturesList || [], name: 'Judges Signatures' },
+  ];
+
+  const history = useHistory();
+  const [title, setTitle] = useState(`Contract id: ${contractId}`);
+
+  const handleClick = (id) => {
+    const routerLink = router.contracts.item.split(':')[0];
+    history.push(`${routerLink}${id}`);
+  };
+  return (
+    <Card
+      title={<Title level={1}>{title}</Title>}
+      extra={!isOneItem ? (
+        <Button link onClick={() => handleClick(contractId)}>More</Button>
```

Comment:
This should be regular link, no? Or clickable with middle mouse button

Reply:
Fixed :-)